minutes/2019-02-05-tooling-group-meeting.md

```diff
@@ -0,0 +1,108 @@
+    - Moving towards a standard would reduce confusion in the community
+  - Refael: why can’t I webpack my backend? It’s not applicable, but it’s confusing
+    - Would ask TC39 to think about JS outside of the browser
+  - Daniel:
+    - Bundling is not TC39 work, it’s W3C stuff
+    - “Import maps” would be necessary (?) for this to work. Web incubator community group, not at TC39 either
+  - Ben: at a meta level there’s a disconnect between how a web browser and node represents files
+    - Finding more common ground
+    - How can we make URIs more palatable to the backend, for example
+    - Nobody wants to provide a URI, they just want to require lodash
+  - Daniel:
+    - Import maps can solve this problem maybe
+  - Chris: import maps? Not TC39?
+  - Daniel: sometimes its unclear where standards should be developed  
+Ben: what’s the best way to have dialog with TC39?
+  Daniel: still trying to figure it out. Proposals are developed in open on GitHub, but this doesn’t always work because the people on the proposals don’t have the full context of the people giving feedback
+    - Daniel wants to talk to them separately and bring feedback back to TC39
+    - But may not be the best idea to have an intermediary
+  - Ben: core node collabs have a node-centric view, there’s userland people using node, there’s browserland and browser vendors… nobody sees eye-to-eye necessarily
+    - How can the group bridge the gaps?
+  - Daniel: we want people prototyping stuff using emerging standards
+  - Ben: like babel. We can reach out to tooling authors
+  - Chris: does TC39 know about esm? (userland module)
+    - Daniel: Yes
+  - Daniel: experimentation like using webpackage would be helpful
+  - Ben: c8 uses v8 coverage, which is experimentation
+    - In the module discussion, it’s frustrating
+    - We want to move forward our ecosystem
+    - Package maps. But can’t move forward without experimental tooling
+  - Daniel: Give feedback to proposals within github issues
+  - Chris: a suggestion box!
+  - Daniel: We thinking about setting up discourse
+    - It can be hard to extract the feedback from angry people
+  - Ben:
+    - The impact of a tool can be more valuable than a discussion
+    - More tools, less conversation!
+  - Daniel:
+    - There’s an import maps polyfill somewhere
+    - Guy Bedford (not present) owns this
+Refael: Give us an idea of where TC39 sees itself in the ecosystem
+  - Daniel:
+    - TC39 sees itself as responsible for the language
+    - But it’s unclear where some things land? Are they in the language?
+    - Please participate in node open standards repo?
+    - Have a call thereafter
+  - Ben:  Maybe regular TC39 rep, but not entire meeting
+  - Daniel: Will try to show at next meeting to keep conversation going
+  
+- Chris: meeting how often? 3wks ok?
+  - Ben: let’s slack more
+  - Chris: maybe go to blessed slack
+  - Rick: yes
+- Rimraf: Sam Roberts (IBM) will comment on rimraf-in-C++ issue
```

Comment:
I thought I did!  Is there a thread I should be watching?

Reply:
this was written before you did